src/SFA.DAS.ApplyService.Web.UnitTests/Services/RoatpTaskListWorkflowServiceTests.cs

```diff
@@ -1,29 +1,53 @@
 ﻿using System;
 using System.Collections.Generic;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Moq;
 using NUnit.Framework;
 using SFA.DAS.ApplyService.Domain.Apply;
 using SFA.DAS.ApplyService.Domain.Entities;
 using SFA.DAS.ApplyService.Web.Configuration;
+using SFA.DAS.ApplyService.Web.Infrastructure;
 using SFA.DAS.ApplyService.Web.Services;
 
 namespace SFA.DAS.ApplyService.Web.UnitTests.Services
 {
     [TestFixture]
     public class RoatpTaskListWorkflowServiceTests
     {
+        private Mock<IQnaApiClient> _qnaApiClient;
+        private Mock<INotRequiredOverridesService> _notRequiredOverridesService;
+        private RoatpTaskListWorkflowService _service;
+        private Guid _applicationId;
+
+        [SetUp]
+        public void Before_each_test()
+        {
+            _applicationId = Guid.NewGuid();
+            _qnaApiClient = new Mock<IQnaApiClient>();
+            _notRequiredOverridesService = new Mock<INotRequiredOverridesService>();
+            _notRequiredOverridesService.Setup(x => x.GetNotRequiredOverrides(_applicationId))
+                                        .Returns(new List<NotRequiredOverrideConfiguration>());
+            
+            var config = new Mock<IOptions<List<TaskListConfiguration>>>();
+            var logger = new Mock<ILogger<RoatpTaskListWorkflowService>>();
+
+            _service = new RoatpTaskListWorkflowService(_qnaApiClient.Object, _notRequiredOverridesService.Object, config.Object, logger.Object);
+        }
+
         [Test]
         public void Get_SectionStatus_Empty_When_Null_Sequences_Used()
         {
             var expectedResult = string.Empty;
-            var actualResult = RoatpTaskListWorkflowService.SectionStatus(null, null, 0, 0);
+            var actualResult = _service.SectionStatus(_applicationId, 0, 0, null, null);
             Assert.AreEqual(expectedResult, actualResult);
```

Comment:
Would be a tiny bit easier to read these tests if magic numbers became named variables.  Same for all tests in this file

Reply:
Reformatted unit tests so easier to read